modmail/utils/extensions.py

```diff
@@ -34,8 +34,9 @@ def determine_bot_mode() -> int:
     The configuration system uses true/false values, so we need to turn them into an integer for bitwise.
     """
     bot_mode = 0
+    _config = config()
```

Comment:
bot.config

Reply:
bot is not available in this context.

Reply:
I do realise this likely a future problem, so I'll add some comments.